packages/block-editor/src/components/use-setting/index.js

```diff
@@ -49,6 +49,18 @@ const deprecatedFlags = {
 	'spacing.customPadding': ( settings ) => settings.enableCustomSpacing,
 };
 
+const filterColorsFromCoreOrigin = ( path, colors ) => {
```

Comment:
In themes that don't set a palette, we are not showing any color as available 
![image](https://user-images.githubusercontent.com/11271197/118718990-ceebf800-b81f-11eb-9bf3-4bebb20d920b.png).


We should just exclude core colors from the UI if there are not theme or user colors I guess a hierarchy approach would also solve this issue.

Reply:
Oh, missed that use case. Fixed at https://github.com/WordPress/gutenberg/pull/31669/commits/172af8da72b3b6a2604fe5a56bd75592c76cda61